core/tests/CMakeLists.txt

```diff
@@ -1,8 +1,6 @@
 # core/tests/CMakeLists.txt
 # Core tests build configuration
 
-include(CTest)
-
```

Comment:
Whats the reasoning behind this?

Reply:
idk, it didn't look necessary and everything still work after removing it. if I shouldn't have removed it lmk why

Reply:
I thought it was necessary for add_tests /shrug